MediaBrowser.Api/Library/LibraryStructureService.cs

```diff
@@ -281,24 +281,14 @@ public void Post(RenameVirtualFolder request)
             }
             finally
```

Comment:
Now that we are not re-scanning the library on rename, it may be necessary to add a `catch` block here and alert the user if the rename failed. Seems like something that should have already been here, but maybe the assumption was that if it failed the library re-scan would throw an error?